_posts/2016-08-27-mouse-dna-magnetic-fields.md

```diff
@@ -0,0 +1,48 @@
+We have since harnessed this unified electromagnetism to provide for the energy needs of billions around the globe.
+However, it is thought that prolonged exposure to ambient electromagnetic (EM) fields can affect certain cells in the body, causing a variety of cancers.
+For this reason, several countries have adopted limits for EM exposure, with most European countries limiting fields to 0.1&nbsp;mT ([milliteslas](https://en.wikipedia.org/wiki/Tesla_(unit))).
+
+Researchers tested the effects of EM fields on biological cells in a controlled environment by exposing two groups of mice to low-frequency fields of 0.1&nbsp;mT and 1.0&nbsp;mT respectively over eight weeks, while a third group of mice living in similar conditions but without the EM exposure served as control.
+The frequency chosen for the research was 50&nbsp;Hz, which is the frequency in most countries of the transmission of electricity via power lines.
+The same researchers previously found that an eight-week exposure to a higher field of 1.5&nbsp;mT caused unrepaired DNA damage in mice.
+In both these studies, the scientists used breaks in the DNA strands in the cell nuclei as a measure of DNA damage.
+In the latest study, they also looked for signs that DNA inside mitochondria was being produced at lower rates than usual.
+
+<!--break-->
+
+Previous research has suggested that extended exposure to EM fields damages cells that are in some way related to the transport, reabsorption or storage of iron.
+The researchers therefore studied how cells involved in these activities located in the brain, the kidney and the liver were affected by EM exposure.
+
+![“DNA com GGN”]({{ site.url }}/images/2016-08-27-mouse-dna-magnetic-fields.jpg)
+
+> _“DNA com GGN”. [Image](https://commons.wikimedia.org/wiki/File:DNA_com_GGN.jpg) by Nogas1974 / [CC BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/)_
+
+Three sets of mice were kept in identical conditions differing only in the EM fields inside their cages.
+For eight weeks, one group of 20 mice was exposed to 0.1&nbsp;mT and the second (also 20) to 1.0&nbsp;mT, while the control group of 16 had no EM exposure beyond that of the earth's magnetic field.
+After eight weeks, cells from ten mice from each group were analysed for signs of DNA damage.
+
+The researchers performed a so-called "blind" analysis, which means they didn't know which mice the cells came from, so that they didn't bias the results in any way with knowledge about the EM-exposure levels involved.
+They looked for specific characteristics in the cells that might show signs of unrepaired DNA damage and then ran statistical tests to determine whether any differences between EM-exposed mice and unexposed mice were statistically significant.
```

Comment:
They looked for signs of DNA damage in the cells that could be tied to the radiation levels each group of mice was exposed to.
leave the statistical test stuff out.